windows/RNSketchCanvas/RNSketchCanvasModule.h

```diff
@@ -34,21 +34,24 @@ namespace winrt::RNSketchCanvas
       std::function<void(JSValue, JSValue)> callback
     ) noexcept
     {
-      XamlUIService uiService = XamlUIService::FromContext(reactContext.Handle());
-      auto sketchCanvasInstance = uiService.ElementFromReactTag(tag).as<winrt::RNSketchCanvas::implementation::RNSketchCanvasView>();
-      IAsyncOperation<winrt::hstring> asyncOp = sketchCanvasInstance.get()->getBase64(type, transparent, includeImage, includeText, cropToImageSize);
-      asyncOp.Completed([=](IAsyncOperation<winrt::hstring> const& sender, AsyncStatus const asyncStatus)
-        {
-          if (asyncStatus == AsyncStatus::Error)
+      reactContext.UIDispatcher().Post([=]()
+      {
+        XamlUIService uiService = XamlUIService::FromContext(reactContext.Handle());
+        auto sketchCanvasInstance = uiService.ElementFromReactTag(tag).as<winrt::RNSketchCanvas::implementation::RNSketchCanvasView>();
+        IAsyncOperation<winrt::hstring> asyncOp = sketchCanvasInstance.get()->getBase64(type, transparent, includeImage, includeText, cropToImageSize);
```

Comment:
I don't think you need the `.get()` part, you should be able to say `sketchCanvasInstance.getBase64(...)`

Reply:
Thank you for the comment.
You're right. I remember needing that get before or I'd get some build error, but looks like I'm not getting those errors anymore, so that's good ;)
I'll remove it.